# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/self-test/delazification-mode-00.js
# Commit: 36780658b33a
# Full Hash: 36780658b33a050e82385161eab2e5e2e5e90064
# Author: Tooru Fujisawa <arai_a@mac.com>
# Date: 2024-12-05 09:58:04
# Regressor Bug: 1933681
# File Overlap Count: 1
# Description:
#   Bug 1933681 - Part 2: Rewrite concurrent delazification's input/output based on InitialStencilAndDelazifications. r=nbp
#   
#   The concurrent delazification internal is still using CompilationStencilMerger.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D230388
# ==============================================================================

diff -r 31d3f2901db9 -r 36780658b33a js/src/jit-test/tests/self-test/delazification-mode-00.js
--- a/js/src/jit-test/tests/self-test/delazification-mode-00.js	Wed Dec 04 23:44:33 2024 +0000
+++ b/js/src/jit-test/tests/self-test/delazification-mode-00.js	Wed Dec 04 23:44:33 2024 +0000
@@ -6,8 +6,8 @@
   }
 
   // Wait is skipped as the source is not registered in the stencil cache.
-  waitForStencilCache(foo);
-  assertEq(isInStencilCache(foo), false);
+  waitForDelazificationOf(foo);
+  assertEq(isDelazificationPopulatedFor(foo), false);
 `;
 
 const options = {